dags/glam_glean_imports.py

```diff
@@ -0,0 +1,123 @@
+"""
+Desktop ETL for importing glean data into GLAM app
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from operators.gcp_container_operator import GKENatPodOperator
+from operators.task_sensor import ExternalTaskCompletedSensor
+from airflow.models import Variable
+from airflow.operators.subdag_operator import SubDagOperator
+
+from glam_subdags.extract import extracts_subdag, extract_user_counts
+from glam_subdags.histograms import histogram_aggregates_subdag
+from glam_subdags.general import repeated_subdag
+from glam_subdags.generate_query import generate_and_run_desktop_query
+from utils.gcp import bigquery_etl_query, gke_command
+from utils.tags import Tag
+
+default_args = {
+    "owner": "akommasani@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 10, 22),
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "akommasani@mozilla.com",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=30),
+}
+
+
+
+tags = [Tag.ImpactTier.tier_2]
+
+dag = DAG(
+    "glam_glean_imports",
+    default_args=default_args,
+    schedule_interval="0 5 * * *",
+    doc_md=__doc__,
+    tags=tags,
+)
+
+
+# Make sure all the data for the given day has arrived before running.
+wait_for_fenix = ExternalTaskCompletedSensor(
+    task_id="wait_for_fenix",
+    external_dag_id="glam_fenix",
+    external_task_id="pre_import",
+    execution_delta=timedelta(hours=3),
+    check_existence=True,
+    mode="reschedule",
```

Comment:
Does this task ever fail? I'm just wondering how do we know it's actually being executed instead of just constantly being rescheduled?